exercises/concept/secrets/secrets.py

```diff
@@ -0,0 +1,14 @@
+def secret_add(secret):

```

Comment:
Docstrings should be added for all functions here.  Please see [this exercise](https://github.com/exercism/python/blob/main/exercises/concept/meltdown-mitigation/conditionals.py) for an example of the format.